src/main/resources/db/postgres/V1_11__CV.sql

```diff
@@ -0,0 +1,53 @@
+CREATE TABLE BRUKER_CV
+(
+    AKTOERID              VARCHAR(20) NOT NULL,
+    HAR_DELT_CV           boolean DEFAULT false,
+    SISTE_MELDING_MOTTATT TIMESTAMP,
+
+    PRIMARY KEY (AKTOERID)
+);
+
+
+DROP VIEW BRUKER;
+CREATE VIEW BRUKER AS
+SELECT OD.AKTOERID,
+       OD.OPPFOLGING,
+       OD.STARTDATO,
+       OD.NY_FOR_VEILEDER,
+       OD.VEILEDERID,
+       OD.MANUELL,
+       OBA.FODSELSNR,
```

Comment:
ser ikke ut til at du bruker den nyeste versjonen av viewet som inneholder kjonn og fodselsdag

Reply:
Ja, ble to migrasjoner på V1_11 etter merge med dev, rettet opp nå :)